lib/cli-engine.js

```diff
@@ -338,6 +338,74 @@ function processPath(extensions) {
     };
 }
 
+/**
+ * create a md5Hash of a given string
+ * @param  {string} str the string to calculate the hash for
+ * @returns {string}    the calculated hash
+ */
+function md5Hash(str) {
+    return crypto
+        .createHash("md5")
+        .update(str)
```

Comment:
@sindresorhus didn't know that, what's the issue of the lack of encoding? just curious.